build does not work anymore ...

just did a cleanall and build and it does not work anymore ..
I'll try to get an old version online again

2015-09-01 17:21:37 (2.33 MB/s) - ‘.build/externs/jquery-1.9.js’ saved [51584/51584]

touch .build/externs/jquery-1.9.js
./node_modules/openlayers/node_modules/.bin/closure-util build build.json .build/build.js
info closure-util Reading build config
info closure-util Getting Closure dependencies
info closure-util Compiling 431 sources
ERR! compile .build/externs/angular-1.4.js:2234: ERROR - Bad type annotation. missing closing )
ERR! compile  * @typedef {function(Function, number=, boolean=, ...*=):!angular.$q.Promise}
ERR! compile                                                        ^
ERR! compile
ERR! compile
ERR! compile 1 error(s),
ERR! compile 0 warning(s)
ERR! compile
ERR! closure-util Process exited with non-zero status, see log for more detail: 1
make: *** [.build/build.js] Error 1
[root@workshop3-local geoportailv3]#

--- a/CONST_Makefile
+++ b/CONST_Makefile
@@ -513,22 +513,22 @@ $(OUTPUT_DIR)/build.css: $(LESS_FILES) .build/node_modules.timestamp
 
 .build/externs/angular-1.4.js:
 	mkdir -p $(dir $@)
-	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/master/contrib/externs/angular-1.4.js
+	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/5ff0611/contrib/externs/angular-1.4.js
 	touch $@
 
 .build/externs/angular-1.4-q_templated.js:
 	mkdir -p $(dir $@)
-	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/master/contrib/externs/angular-1.4-q_templated.js
+	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/5ff0611/contrib/externs/angular-1.4-q_templated.js
 	touch $@
 
 .build/externs/angular-1.4-http-promise_templated.js:
 	mkdir -p $(dir $@)
-	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/master/contrib/externs/angular-1.4-http-promise_templated.js
+	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/5ff0611/contrib/externs/angular-1.4-http-promise_templated.js
 	touch $@
 
 .build/externs/jquery-1.9.js:
 	mkdir -p $(dir $@)
-	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/master/contrib/externs/jquery-1.9.js
+	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/5ff0611/contrib/externs/jquery-1.9.js
 	touch $@
 
 package.json:

